rpc_jobs/standard_job_premerge.yml

```diff
@@ -0,0 +1,169 @@
+- project:
+    # This is the unique name to designate to the
+    # project. The name should relate to the git
+    # repository name for easier identification.
+    name: "sample"
+
+    # This is the URL for the project's repository
+    # which contains the scripts which will be
+    # executed.
+    repo_url: "https://github.com/rcbops/sample"
+
+    # This is the whitelist for branch names which
```

Comment:
I think it would be useful to mention that this list is only for PRs.

Reply:
Done.